build.gradle

```diff
@@ -4,7 +4,7 @@ plugins {
 }
 
 group 'com.github.amibiz'
-version '1.13.213'
+version '1.13.214'
```

Comment:
Just a convention I use, can you please change to `1.13.222` (to match the branch number).

see: https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html?from=jetbrains.org#intellij-platform-based-products-of-recent-ide-versions